doc/tutorials/acrn-secure-boot-with-grub.rst

```diff
@@ -0,0 +1,177 @@
+.. _how-to-enable-acrn-secure-boot-with-grub:
+
+Enable ACRN Secure Boot with GRUB
+#################################
+
+This document shows how to enable ACRN secure boot with GRUB including:
+
+  -  ACRN Secure Boot Sequence
+  -  Generate GPG Key
+  -  Setup Standalone GRUB EFI Binary
+  -  UEFI Secure Boot
+
+  Validation Environment:
+	Hardware Platform: INTEL WHL
+	ACRN Scenario: Industry
+	Service VM: Ubuntu 18.04
+	GRUB: 2.02
+
+Note: Please make sure you have alternative ways to restore GRUB when you got stuck.
+
+ACRN Secure Boot Sequence
+*************************
+
+  ACRN can be booted by Multiboot compatible bootloader, following diagram illustrates the boot sequence of ACRN with GRUB:
+
+     .. image:: images/acrn-secureboot-flow.png
+        :align: center
+        :width: 600px
+
+
+  Refer to :ref:`using_grub` for details to enable GRUB on ACRN.
+
+  From point view of secureboot:
+	- UEFI firmware verifies Shim/GRUB
+	- GRUB verifies ACRN, Service VM kernel and Prelaunch VM kernel
+	- SOS kernel verifies DM and UOS OVMF thru ACRN DM
+	- UOS OVMF starts the guest side verified boot process
+
+  This document provides guide to enable verification function of GRUB to verify ACRN binaries(such acrn.bin, service VM bzImage and Prelaunch VM kernel image if it is present).
+
+
+Generate GPG Key
+****************
+
+  GRUB supports loading of GPG signed files only if digital signatures enabled.
+  Example to generate GPG signing key::
+  $mkdir --mode 0700 keys
```

Comment:
```suggestion
  $ mkdir --mode 0700 keys
```
Do this too for the lines below.

I suspect this code-block formatting is not rendered as expected, have you verified the final rendering (using `make doc`)?

Reply:
@gvancuts my local make doc does not work,  not see error reported from CI /build doc, are they same?

//my local build doc err, any hint?

make[1]: Entering directory '/home/yh/workspace/acrn/acrn-hypervisor/doc'
Makefile:60: recipe for target 'html' failed
make[1]: [html] Error 2 (ignored)

New errors/warnings found, please fix them:
==============================================


Extension error:
Could not import extension sphinx.builders.applehelp (exception: cannot import name 'RemovedInSphinx40Warning')
Makefile:60: recipe for target 'html' failed
make[1]: *** [html] Error 1
make[1]: Leaving directory '/home/yh/workspace/acrn/acrn-hypervisor/doc'
Makefile:184: recipe for target 'doc' failed
make: *** [doc] Error 2